.ci/mac_before_script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+echo 'running mac before_scripts'
+mkdir -p ~/.gnupg
+touch ~/.gnupg/gpg.conf
+sudo chown travis ~/.gnupg/gpg.conf
+chmod 0600 ~/.gnupg/gpg.conf
+gpg --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```

Comment:
I take it these keys are needed for something? This script could use documentation.